data-es6.js

```diff
@@ -3570,8 +3591,9 @@ exports.tests = [
         var obj = { foo: 1, bar: 2 };
         var iterator = Reflect.enumerate(obj);
 
+        var passed = Symbol.iterator in item;
```

Comment:
This line can't really be added because it unnecessarily requires that the engine also support a publicly available `Symbol.iterator`, a strongly related but nonetheless distinct feature. It's still possible for an engine to support `Reflect.enumerate` without _yet_ exposing the `@@iterator` symbol as `Symbol.iterator`.